### DISCUSSION TITLE
Refresh modal content?

### DISCUSSION THREAD

**Discussion by RPotkowaVegasystems** [filamentphp/filament#11439] [Help]
### Package

Panel builder

### Package Version

v3.2.31

### How can we help you?

I have a Table where i can create new Content inside a modal. Inside there is a Repeater with and custom Action where i can create new Repeater Content. But my Action opens a new modal, where i can create the Repeater Content. This works fine, i can enter the new Content/Values and and Hit the Save Button. After that the modal close, the values are stored in the DB and first modal open again.That works. 

But my Problem is, that the new Content/Values are not showing in the Repeater Elements. 

Is it possible, to refresh the first modal, that also my new Content/Values are shown?

**Comment by leandrocfe** [comment]
Maybe forcing it with `mountUsing` ?
https://filamentphp.com/docs/3.x/actions/modals#executing-code-when-the-modal-opens

**Reply by RPotkowaVegasystems** [reply]
I fixed it with a small workaround instead of refresh of the whole modal.
I used: action()->after().


`
->after(function (Set $set)  {
                        $set('myField', $newValuesArray);
                    })
`
